packages/eslint-plugin-eds/src/rules/no-h-tags.ts

```diff
@@ -0,0 +1,31 @@
+import type {Rule} from 'eslint';
+
+const getFailureMessage = (name: string) => {
+  return `Please use the <Heading> component from the @chanzuckerberg/eds repo instead of an <${name}> tag 💕`;
+};
+const hTags = {
+  h1: true,
+  h2: true,
+  h3: true,
+  h4: true,
+  h5: true,
+  h6: true,
+};
+
+const rule: Rule.RuleModule = {
+  create(context) {
+    return {
+      JSXElement(node: any) {
```

Comment:
not sure how to type `node` here

Reply:
@jinlee93 [`JSXOpeningElement`](https://babeljs.io/docs/en/babel-types#jsxelement) ?

Reply:
Looks like the eslint RuleModule types don't really know about the JSX stuff 🤔 Which is weird.

@jinlee93 can you suppress the typescript-eslint/no-explicit-any rule here? That way the warnings don't show up locally and github won't keep posting comments on PRs in this file?